requirements.txt

```diff
@@ -0,0 +1,6 @@
+pybind11 >=2.5.0
```

Comment:
So this directly blocks us from being able to use the pip/conda version to build the rest of the pipelines, right?  It's a newer pybind11 than ndarray supports, and maybe some of our other code.

Reply:
On the plus side this package must be compatible with the new pybind11. This was an arbitrary cut and paste from the pybind11 documentation. Since it worked with 2.5 and we know it builds with the stack version it's not a problem to change it.